File: javascript_wdio/caps_options/test/specs/test_scenario/exampleTest.js
const ta = require('trueautomation-helper').ta;

describe('TrueAutomation.IO page WebdirverIO + TrueAutomation', () => {
  it('Test example', () => {
    browser.setTimeout({ 'implicit': 5000 });

    browser.url('https://trueautomation.io');
    $(ta('loginBtn', "//div[./span[text()='Login']]")).click();
    $(ta('signUpBtn', 'div.sign-up-container > a')).click();
    $(ta('emailFld', "[name='email']")).setValue('your@gmail.com');
  });
});

File: capybara_rspec/browser_options/spec/test_scenario/example_test.rb
require 'spec_helper'

feature 'TrueAutomation.IO capybara example' do
  scenario 'Test example' do
    visit 'https://trueautomation.io/'

    find(:xpath, ta('trueautomationio:home:login_btn', "//a[contains(., 'Login')]")).click
    find(:css, ta('trueautomationio:signin:signup_btn', 'div.sign-up-container > a')).click
    fill_in ta('trueautomationio:signup:email', 'email'), with: 'your@gmail.com'
    #
    # find(:xpath, ta('button1')).click
    # sleep 3
    # find(:css, ta('button2')).click
    # sleep 3
    # fill_in ta('field'), with: 'your@gmail.com'
    # sleep 3
  end
end
